themes/Delicious/foundation/typography.scss

```diff
@@ -90,8 +114,47 @@ $font-size-data: (
   display-small: (
     base: rem(16px),
     large-screen: rem(20px)
+  ),
+  heading-h1:(
+    base:rem(40px)
+  ),
+  heading-h2:(
+    base:rem(32px)
+  ),
+  heading-h3:(
+    base:rem(24px)
+  ),
+  heading-h4:(
+    base:rem(16px)
+  ),
+  heading-subheading:(
+    base:rem(16px)
+  ),
+  textbody-largetext:(
+    base:rem(16px)
+  ),
+  textbody-medtext:(
+    base:rem(14px)
+  ),
+  textbody-smalltext:(
+    base:rem(13px)
   )
+
 );
+ 
+
+$font-weight-data: (
+  bold:(
+    base:700
+  ),
+  semi-bold:(
+    base:600
+  ),
+  regular:(
```

Comment:
@Farhan-emgage use "normal" instead of "regular". That way it corresponds to the css value.